Health/src/main/java/com/sciencesquad/health/MainActivity.java

```diff
@@ -34,6 +43,13 @@ protected void onCreate(Bundle savedInstanceState) {
 
 		NavigationView navigationView = (NavigationView) findViewById(R.id.nav_view);
 		navigationView.setNavigationItemSelectedListener(this);
+
+		RealmConfiguration defaultConfig = new RealmConfiguration.Builder(this)
```

Comment:
This is a TERRIBLE idea since what you're saying is "Whenever this Activity is created, reconfigure and create a new `NutritionModule`". Serious flaw: the `Activity` will be recreated whenever you leave it for a while and come back (i.e. multitasking drawer).